src/Codeception/Test/Metadata.php

```diff
@@ -200,7 +200,11 @@ public function setParamsFromAnnotations($annotations): void
 
```

Comment:
I noticed that after `array_merge_recursive`, `$this->params['skip']` contains array `[null, 'message']`